package.json

```diff
@@ -39,18 +39,22 @@
     "test": "run-s test:backend test:frontend",
     "test:backend": "cd ./backend && npm run test",
     "test:frontend": "cd ./frontend && npm run test",
+    "test:e2e": "cd ./frontend && npm run test:e2e",
+    "test:accessibility": "cd ./frontend && npm run test:accessibility",
+    "test:integration": "cd ./frontend && npm run test:integration",
+    "test:unit": "cd ./frontend && npm run test:unit",
     "test:fix": "run-s test:fix-backend test:fix-frontend",
     "test:fix-backend": "cd ./backend && npm run test:fix",
-    "test:fix-frontend": "cd ./frontend && npm run test:fix",
-    "test:integration": "jest --roots=./tests",
-    "test:integration-dev": "npm run build; jest --roots=./tests --watchAll",
-    "test:integration-fix": "npm run build; jest --roots=./tests --update-snapshot"
+    "test:fix-frontend": "cd ./frontend && npm run test:fix"
   },
   "dependencies": {
     "dotenv": "^8.2.0",
     "dotenv-expand": "^5.1.0",
     "jest": "^26.6.3",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.2.1"
+  },
+  "devDependencies": {
```

Comment:
Would this not be optional dependencies? Playwright is a problem for downstream installs...

Reply:
ill move it